src/app/contribution-graph/contribution-graph.component.ts

```diff
@@ -45,4 +48,25 @@ export class ContributionGraphComponent implements OnInit {
     return Array.from(Array(row).keys());
   }
 
+  calcDays(startDateString: string) {
```

Comment:
Probably better to use a more informative function name. Just `calcDays()` makes it difficult to understand that it should return the week-day labels.